components/Sections/SectionsBuilder.tsx

```diff
@@ -2,11 +2,13 @@ import prisma from "@/lib/prisma";
 import { ImageTextColumnsBuilder } from "./ImageTextColumns/ImageTextColumns";
 import { RichTextBuilder } from "./RichText/RichText";
 
-interface PageSectionsBuilder {
+interface PageSectionsBuilderProps {
   pagesId: number;
 }
```

Comment:
Good call! Also we should probably rename `pagesId` to `pageId` since it's referring to one group (even though in the Prisma schema it's the former).